src/status_im/extensions/registry.cljs

```diff
@@ -0,0 +1,114 @@
+(ns status-im.extensions.registry
+  (:refer-clojure :exclude [list])
+  (:require [clojure.string :as string]
+            [pluto.utils :as utils]
+            [pluto.reader.hooks :as hooks]
+            [re-frame.core :as re-frame]
+            [status-im.accounts.update.core :as accounts.update]
+            [status-im.i18n :as i18n]
+            [status-im.utils.fx :as fx]
+            [clojure.set :as set]))
+
+(fx/defn update-hooks
+  [{:keys [db] :as cofx} extension-key]
+  (let [{:keys [dev-mode?] :as account} (get db :account/account)
+        {:keys [active? hooks]} (get-in account [:extensions extension-key])]
+    (apply utils/merge-fx cofx
```

Comment:
looks complex, it's difficult to read and understand what happens here

Reply:
simplified it.
however, this function was simply moved from `pluto` here and originally contained almost no changes.